saleor/product/models.py

```diff
@@ -392,6 +403,21 @@ def __str__(self):
     def quantity_available(self):
         return max(self.quantity - self.quantity_allocated, 0)
 
+    def get_costs_data(self):
+        costs = []
+        margins = []
+        if self.cost_price:
+            cost = self.cost_price
+            costs.append(cost)
+            price = self.variant.get_price_per_item()
+            margin = price - cost
+            percent = round((margin.gross / price.gross) * 100, 0)
+            margins.append(percent)
+        costs = sorted(costs, key=lambda x: x.gross)
+        margins = sorted(margins)
+        print margins
+        return {'costs': costs, 'margins': margins}
```

Comment:
Is there a benefit to returning a dict here instead of a tuple?

Reply:
I think that is more readible because we returns here two different things.

Reply:
Hmmm i will split this to two functions